RetrospectiveExtension.Frontend/index.tsx

```diff
@@ -6,13 +6,9 @@ import { init as sdkInit }  from 'azure-devops-extension-sdk';
 import { isHostedAzureDevOps } from './utilities/azureDevOpsContextHelper';
 import { getProjectId } from './utilities/servicesHelper';
 import './css/main.scss';
+import { appInsights } from './utilities/external/telemetryClient';
 
 import FeedbackBoardContainer, { FeedbackBoardContainerProps } from './components/feedbackBoardContainer';
```

Comment:
Why did you remove `initializeIcons()` method?

Reply:
added it back, it was between two lines of TODO and must have been deleted by mistake

Reply:
good catch! :)